src/bytes/complete.rs

```diff
@@ -183,6 +390,27 @@ where
   }
 }
 
+/// Matches a byte string with escaped characters.
+///
+/// * The first argument matches the normal characters (it must not accept the control character),
+/// * the second argument is the control character (like `\` in most languages),
+/// * the third argument matches the escaped characters
+///
+/// # Example
+/// ```
+/// # #[macro_use] extern crate nom;
+/// # use nom::{Err, error::ErrorKind, Needed};
+/// # use nom::character::complete::digit1;
+/// use nom::bytes::complete::escaped;
+/// use nom::character::complete::one_of;
+/// let esc = |s: &'static str| {
+///   escaped::<&str, (_, ErrorKind), _, _, _, _>(digit1, '\\', one_of("\"n\\"))(s)
```

Comment:
maybe write the function like this to avoid all those generic parameters:
```rust
fn esc(s: &str) -> IResult<&str, &str> {
  escaped(digit1, '\\', one_of("\"n\\"))(s)
}
```